src/component/noteCard/NoteCard.jsx

```diff
@@ -14,7 +15,7 @@ export const NoteCard = ({ item, archive }) => {
   const { setNotesList } = useNotesList();
 
   const { note, displayModal, noteDispatch } = useNoteInputContext();
-
+  const { archivesList, setArchivesList } = useArchivesList();
   useEffect(() => {}, [displayModal]);
```

Comment:
Suggestion: As per my understanding this useEffect has no use, correct me if I am wrong , if not then you can remove it

Reply:
Yup you are right